var/spack/repos/builtin/packages/py-cymem/package.py

```diff
@@ -13,7 +13,9 @@ class PyCymem(PythonPackage):
     homepage = "https://github.com/explosion/cymem"
     pypi = "cymem/cymem-2.0.3.tar.gz"
 
+    version("2.0.7", sha256="e6034badb5dd4e10344211c81f16505a55553a7164adc314c75bd80cf07e57a8")
     version("2.0.3", sha256="5083b2ab5fe13ced094a82e0df465e2dbbd9b1c013288888035e24fd6eb4ed01")
 
     depends_on("py-setuptools", type="build")
     depends_on("py-wheel@0.32.0:0.32", type="build")
```

Comment:
This can be shortened to
```suggestion
    depends_on("py-wheel@0.32", type="build")
```

Reply:
Where do you even see that wheel 0.32 is required? If a specific version isn't required, you can just remove this dependency, it's already added by the base class.

Reply:
[Version 2.0.3](https://files.pythonhosted.org/packages/ce/8d/d095bbb109a004351c85c83bc853782fc27692693b305dd7b170c36a1262/cymem-2.0.3.tar.gz) `setup.py:124`. Should I add a `when="@2.0.3"` to this?

Reply:
Yes, please do